accname/name/comp_name_from_content_menuitem.html

```diff
@@ -0,0 +1,107 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Name From Content</title>
+  <meta charset="utf-8">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <style type="text/css">
+     /*
+
+      Since the AccName spec is in flux about whitespace joiners, and the implementations don't match,
+      normalize the whitespace (with extra leading/trailing) space.
+
+      No point in having the same in-flux spec change cause failures the remaining unrelated tests.
+
+      See more here:
+      https://github.com/w3c/accname/issues/205
+      https://github.com/w3c/accname/projects/1#card-42288231
+
+    */
+  </style>
+</head>
+<body>
+
+<!--TODO: Add test description-->
+<!--  TODO: Add tests for empty strings, empty alts, hidden elements (but not aria-hidden)-->
+  <h1>Named from Content for Anchor Menuitems</h1>
+  <ul role="menu">
+    <li>
```

Comment:
Sometimes these in-betweener elements (the `<li>`) cause differences in the tree. If so, you can always explicitly define the role as "none" to normalize the tests... If you find some problem, please workaround it here and file it as a new issue so we can add a new test in /wai-aria or /html-aam